src/api/Configuration.php

```diff
@@ -92,8 +92,14 @@ public function isAutofinishSpansEnabled()
     public function getSamplingRate()
     {
         // DD_SAMPLING_RATE is deprecated and will be removed in 0.40.0
```

Comment:
Looks like it didn't get removed in 0.40.0 after all! 😆

Reply:
Yep, it's a simple alias currently :-)